modules/ROOT/pages/provisioning-raspberry-pi4.adoc

```diff
@@ -76,7 +78,6 @@ Now take the USB/SD card and attach it to the RPi4 and boot.
 
 TIP: It can take some time to boot, especially if the disk is slow. Be patient. You may not see anything on the screen for 20-30 seconds.
 
-
```

Comment:
Do we need to remove these extra line breaks? When viewing the file plain text I like having the extra break before the start of a new section. Maybe the issue here is consistency?

Reply:
They didn't seem needed to me; I don't think the rendering changes, I'll confirm.

If you have a preference for the additional breaks, it's fine to leave them in.

Reply:
Render changed slightly; I added them back in

Reply:
reviewed!